elixir/config/runtime.exs

```diff
@@ -109,6 +109,12 @@ if config_env() == :prod do
   config :domain, Domain.Auth.Adapters.Okta.Jobs.SyncDirectory,
     enabled: compile_config!(:background_jobs_enabled)
 
+  config :domain, Domain.Auth.Adapters.JumpCloud.Jobs.SyncDirectory,
```

Comment:
@bmanifold I noticed JumpCloud was missing here - was this intentional?

Reply:
No, this was likely an oversight.  Thanks for catching this.

I am a little confused how this was working, but I remember testing it in staging.